common/src/Microsoft.Azure.IIoT.Agent.Framework/src/Agent/Default/WorkerSupervisor.cs

```diff
@@ -110,12 +110,12 @@ public class WorkerSupervisor : IWorkerSupervisor, IDisposable {
             while (true) {
                 var workers = _agentConfigProvider.Config?.MaxWorkers ?? kDefaultWorkers;
                 while (_instances.Count < workers) {
-                    _logger.Information("Creating new worker...");
+                    _logger.Verbose("Creating new worker...");
```

Comment:
>Verbose [](start = 28, length = 7)

this is typically only happening once during runtime, I think we can keep it to Information level, same below

Reply:
The reasoning here is that it's a bit redundant:
1. Creating new worker...
2. Starting worker 'StandalonePublisher_1'...
3. Worker: StandalonePublisher_1, start processing new job: Standalone_test_publisher, mode: "Active"